client/me/purchases/utils.js

```diff
@@ -9,8 +9,9 @@ import page from 'page';
 import analytics from 'lib/analytics';
 import paths from './paths';
 
+// TODO: Remove these property-masking functions in favor of selectors
 function getPurchase( props ) {
-	return props.selectedPurchase.data;
```

Comment:
This `utils` file doesn't work on the internal structure of the purchase anymore, it only works on the props. i think we could remove this file and access the props directly as it just make the code harder to follow.